src/app/clusters/level-control/level-control.cpp

```diff
@@ -992,23 +993,27 @@ void emberAfOnOffClusterLevelControlEffectCallback(EndpointId endpoint, bool new
 
 void emberAfLevelControlClusterServerInitCallback(EndpointId endpoint)
 {
+    EmberAfLevelControlState * state = getState(endpoint);
+    state->minLevel                  = EMBER_AF_PLUGIN_LEVEL_CONTROL_MINIMUM_LEVEL;
```

Comment:
Again, what guarantees `state` is not null?

Reply:
can a cluster InitCallback be called for an endpoint that does not exist? I was under the impression it could not happen.

Again I can add the check if you prefer

Reply:
Again, the way things are set up right now `stateTable` only knows about static endpoints with the level control cluster on them.  But the init callback will absolutely be called for dynamic endpoints that have the cluster.

Reply:
Sure, i will add the check. However, there is a significant amount of work for multiple clusters to support dynamic endpoint. We had a few discussions before and it is not a trivial task to support it.

Reply:
Sure.  I'm just asking for it to not crash...